### ISSUE TITLE
Training loss appears nan

### ISSUE THREAD

**Issue by DicHui** [wongzbb/DiffMa-Diffusion-Mamba#18]
Thank you very much for the open source！I found that the loss sometimes becomes nan during training, and it seems that there are also nan values in the .npy data you provided. Is this reasonable? Do I need to clean the data?

<img width="750" height="426" alt="Image" src="https://github.com/user-attachments/assets/749e49ea-fe40-45ff-9e8e-6212f77f57ce" />
